ci-operator/config/operator-framework/operator-registry/operator-framework-operator-registry-release-4.9.yaml

```diff
@@ -3,37 +3,21 @@ base_images:
     name: "4.9"
     namespace: ocp
     tag: base
-  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+  ocp_builder_rhel-8-golang-1.16-openshift-4.8:
     name: builder
     namespace: ocp
-    tag: rhel-8-golang-1.15-openshift-4.7
-  ocp_builder_rhel-8-golang-1.15-openshift-4.8:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-1.15-openshift-4.8
-  ocp_builder_rhel-8-golang-openshift-4.6:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-openshift-4.6
-  ocp_builder_rhel-8-golang-openshift-4.6-clang:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-openshift-4.6-clang
-  ocp_builder_rhel-8-golang-openshift-4.7:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-openshift-4.7
```

Comment:
This is unrelated, but I wonder if we should remove the EOL branch configurations, like the 4.2, 4.3, etc. like we're pruning the old image imports here.

Reply:
Possibly? I actually wasn't sure if those were there for some reason or if they've just been brought along because of the way the next-release automation works. Seems like the latter, and I was going to use this PR to see if everything would still pass if those disappear.